zerver/models/groups.py

```diff
@@ -95,6 +98,16 @@ class NamedUserGroup(UserGroup):  # type: ignore[django-manager-missing] # djang
     }
 
     GROUP_PERMISSION_SETTINGS = {
+        "can_add_members_group": GroupPermissionSetting(
+            require_system_group=False,
+            allow_internet_group=False,
+            allow_owners_group=True,
+            allow_nobody_group=True,
+            allow_everyone_group=True,
```

Comment:
`allow_everyone_group` should be false, like with all other administrative permissions.`allow_everyone_group` should be false, like with all other administrative permissions.